packages/web-runtime/src/components/Topbar/ThemeSwitcher.vue

```diff
@@ -36,20 +36,33 @@ export default {
     }
   },
   mounted() {
-    if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
+    const currentTheme = JSON.parse(window.localStorage.getItem('oc_currentTheme'))
```

Comment:
We have a `useLocalStorage` composable, what about using that one?

Reply:
Interesting, let me take a look 🕵🏽 only searched the codebase for `localStorage` and got "inspired" ™️  by what I found

Reply:
Refactored, thanks for the input 🙌🏽